android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -344,8 +394,12 @@ public void onHostResume() {
              * If the local video track was released when the app was put in the background,
              * recreate.
              */
-            if (cameraCapturer != null && localVideoTrack == null) {
-                localVideoTrack = LocalVideoTrack.create(getContext(), isVideoEnabled, cameraCapturer, buildVideoFormat());
+            if (localVideoTrack == null) {
+                if(screenCapturer != null) {
+                    localVideoTrack = LocalVideoTrack.create(getContext(), isScreenShareEnabled, screenCapturer);
+                } else if(cameraCapturer != null) {
```

Comment:
Code formatting:

```suggestion
                } else if (cameraCapturer != null) {
```

This is a reoccurring issue: make sure you run code formatting on the .java files